recipes/ldrb/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set version = "2021.0.1" %}
+
+package:
+  name: ldrb
+  version: {{ version }}
+
+source:
+  url: https://github.com/finsberg/ldrb/archive/v{{ version }}.tar.gz
+  sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+build:
+  number: 0
+  skip: True  # [win or py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - h5py
+    - fenics
+    - mshr
+    - numba
+    - numpy
+    - quaternion
+    - scipy
```

Better to label this `host` and remove anything that's not needed to run `setup.py`:
```suggestion
  host:
    - python
    - pip
```